benchmarking/README.md

```diff
@@ -116,7 +116,7 @@ After you've deployed the GUI, you can use the starter kit. More details will co
 
 2. If the deployed LLM is a Composition of Experts (CoE), specify the desired expert in the corresponding text box and then set the configuration parameters. If the deployed LLM is not a CoE, simply set the configuration parameters.
 
-3. If the deployed LLM is a SambaNova Cloud endpoint, choose in the API type dropdown the `sncloud` option. 
+3. If the deployed LLM is a SambaNova Cloud endpoint, choose in the API type dropdown the `SambaNova Cloud` option. 
```

Comment:
hey @snova-michaelw about this change, we can mask the references to the api types in UI with the labels you proposed. Unfortunately, applying this change to CLI and notebooks is not standard across all kits and applying this change there is not trivial. Please let's stick with the current values in references not related to UI. Let me know if you have any questions

Reply:
Done. I rolled back the change to one that's much more cosmetic. UI still looks great, but underlying function calls are now all to `sncloud` or `sambastudio`. 
In the future: the fact that applying the change "is not trivial" (and I agree with that assertion) is a clear-cut sign that this code base would benefit from some good ol' fashioned dependency inversion/injection here. That would allow us to easily modify/update/add/remove as many API types as we might want in the future without minimal muss and fuss.
I'd all but insist upon this if this were core prod code; since it's "just" a starter kit, there's an argument to be made that the refactorign is more trouble than it's worth. Your call.